Application Lifecycle Management/api-boxes.md

```diff
@@ -84,223 +100,334 @@ GET /services/boxes?ids=05b76b08-5238-4e05-ae5f-8ea8afe00378
 | event | plain | object | Event contained in one of the event lists, each event object contains the  parameters: url, upload_date, length and destination_path. |
```

Comment:
in variables row, change...
List of box variables, each variable object contains the parameters: type, name, value, visibility, required, etc.

Consider this change in some other rows in this page where the attribute 'variables' is referenced.